src/components/profileEdit/ProfileInfo.tsx

```
@@ -116,321 +126,303 @@ export default function ProfileInfo() {
-                                <PasswordInput
-                                    value={newPasswordConfirm}
-                                    onChange={handleNewPasswordConfirmChange}
-                                />
+                    }
+                    {profile.linkUrls.map((url, index) => {
+                        return (
+                            <div key={index} className="flex gap-2 items-center rounded-full bg-gray-100 px-4 py-1">
+                                <span>{url}</span>
+                                <button 
+                                    onClick={() => handleUrlRemove(index)}
+                                    className="text-sub-gray rounded-md text-sm"
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)